### PR TITLE
INTYGFV-15719: SRS should be visible for signed certificates

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

**Review by ChrilleSigerhed — APPROVED** [review]

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.js [hunk 1]
```diff
@@ -167,14 +167,24 @@ angular.module('common').directive('wcSrsPanelTab',
...
+                }
                 if (content.grundData.relation.relationKod === 'FRLANG') {
                     $scope.srs.isForlangning = true;
                     // $scope.srs.extensionFromIntygId = content.grundData.relation.relationIntygsId;
                     $scope.srs.userClientContext = 'SRS_FRL';
                     $scope.srs.extensionPredictionFetched = false;
                 } else {
                     $scope.srs.isForlangning = false;
-                    $scope.srs.userClientContext = 'SRS_UTK';
+                    $scope.srs.userClientContext = $scope.config.intygContext.isSigned ? 'SRS_SIGNED' : 'SRS_UTK';
```
Comment (ChrilleSigerhed) [inline web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.js:187]: Should we add SRS_REH for userClientContext in angular aswell?

**Review by ChrilleSigerhed — COMMENTED** [review]
Great job, now the SrsPanel is populated with srs-data when navigating from RS. After the certificate is loaded, the tab for SRS is not focused/opened, is that something we need to add for angular aswell?

File: web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.js (hunk 1 above)
Comment (mhornfeldt) [reply web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.js:187]: SRS_REH is already added in the readonly part of the code. There are no requirements saying which tab is supposed to be active so I've kept the original placement and logic for active tab.

Comment (mww-sopra-steria) [reply web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.js:187]: I checked how the current Rehabstod/Webcert-solution looks like. When opening a certificate in RS it doesn't display SRS as default either. I thought the same as @ChrilleSigerhed but the solution you made @mhornfeldt seems to be the correct one.

I guess you can just merge this PR as @ChrilleSigerhed has already approved it. :-)

**Pull request merged**